src/components/Alert.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from './Button';
+import { IoIosWarning } from 'react-icons/io';
+import './Alert.scss';
+import { useLocalStorage } from '../common/hooks';
+
+export default function Alert({ children, dismissableKey = false }) {
+  const [dismissed, setDismissed] = useLocalStorage(
+    `alert/${dismissableKey}`,
+    false
```

Comment:
This would mean that the user will never see the warning again once they dismiss it, even if it's been 6 months since they last used our website. Perhaps we can change it to reappear on every dataset?